security-oauth2/src/test/groovy/io/micronaut/security/oauth2/client/clientcredentials/ClientCredentialsConfigurationSpec.groovy

```diff
@@ -54,4 +54,27 @@ class ClientCredentialsConfigurationSpec extends Specification {
         cleanup:
         applicationContext.close()
     }
+
+    void "is is possible to set additionalRequestParams with client-credentials.additional-request-params"() {
+        given:
+        ApplicationContext applicationContext = ApplicationContext.run([
+                'micronaut.security.oauth2.clients.authservermanual.token.url': "http://foo.bar/token",
+                'micronaut.security.oauth2.clients.authservermanual.client-id': 'XXX',
+                'micronaut.security.oauth2.clients.authservermanual.client-secret': 'YYY',
+                'micronaut.security.oauth2.clients.authservermanual.client-credentials.advanced-expiration': '0s',
+                'micronaut.security.oauth2.clients.authservermanual.client-credentials.additional-request-params.audience': 'test',
+        ])
+
+        when:
+        OauthClientConfiguration configuration = applicationContext.getBean(OauthClientConfiguration, Qualifiers.byName("authservermanual"))
+
+        then:
+        noExceptionThrown()
+
+        expect:
+        configuration.getClientCredentials().get().getAdditionalRequestParams().get("audience") == "test"
```

Comment:
You can rewrite this as: 
        
```groovy
configuration.getClientCredentials().get().getAdditionalRequestParams().get("audience") == "test"
```

as: 

```groovy
"test" == configuration.clientCredentials.get().additionalRequestParams.get("audience") 
```

Reply:
done